src/main/scala/io/iohk/ethereum/blockchain/sync/RegularSync.scala

```diff
@@ -23,11 +21,13 @@ trait RegularSyncController {
 
   import Config.FastSync._
 
-  def regularSync(): Receive = {
-
-    case StartSyncing =>
-      askForHeaders()
+  def startRegularSync(): Unit = {
```

Comment:
Should we do here `appStateStorage.fastSyncDone()` as we previously did when handling the `StartRegularSync` message? If that's not the case we are able to run first the regular sync and then the fast sync, which I think we agreed that we shouldn't allow in #90.